elasticsearch/pom.xml

```diff
@@ -70,25 +47,33 @@
     </resources>
     <plugins>
       <plugin>
-        <artifactId>maven-compiler-plugin</artifactId>
-        <configuration>
-          <fork>true</fork>
-          <compilerArgs>
-            <arg>-J-Dproject.version=${project.version}</arg>
-          </compilerArgs>
-        </configuration>
-      </plugin>
-      <plugin>
         <groupId>io.fabric8</groupId>
-        <artifactId>docker-maven-plugin</artifactId>
-        <version>${docker.maven.plugin.version}</version>
+        <artifactId>fabric8-maven-plugin</artifactId>
+        <version>${fabric8.maven.plugin.version}</version>
+        <executions>
+          <execution>
+            <id>resource</id>
+            <goals>
+              <goal>resource</goal>
+              <goal>build</goal>
+            </goals>
+          </execution>
+          <execution>
+            <id>json</id>
+            <phase>none</phase>
```

Comment:
For `<configurations>` one could use `combine.self="override"` to override everything, but this is not available for `<executions>`. Unfortunately this is indeed a bit tricky because of this mixed setup.

The only alternatives to `<phase>none</phase>` I can think of are:

* Dont bind executions in the parent pom.xml at all but always where it is used (e.g. where f-m-p-2 is still used).
* Use different profiles, so that you could both binding IN the parent pom.xml. These profiles could be activated in the child poms by setting a property (e.g. `use.fmp.3`) and a property activation for the profiles. This is probably the best solution.